bw_matchbox/webapp.py

```diff
@@ -437,14 +437,24 @@ def search():
 
     q = flask.request.args.get("q")
     embed = flask.request.args.get("e")
+    json = flask.request.args.get("json")  # Return the data as json
     search_db = flask.request.args.get("database")
 
-    if embed:
+    table_data = bd.Database(search_db or t).search(q, limit=100),
```

Comment:
This is OK for now, but #56 clearly states "Switch from /search to /processes to have one API call to populate tables". I think this is much more sensible in the long run.

Reply:
I don't understand it. What have I do?